libs/hdf-converters/src/jsonix-intermediate-converter.ts

```diff
@@ -0,0 +1,9 @@
+import {JsonixConverter} from './jsonix-converter';
```

Comment:
make a page on the heimdall monorepo called 'hdf converter how-tos' or something like that which links to the saf cli's pages for how to write a mapper / jsonix converter thing